tests/phpunit/tests/theme/globalStylesheet.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * @group themes
+ */
+class Tests_Global_Stylesheet extends WP_UnitTestCase {
+
+	public function test_block_theme_using_variables() {
+		switch_theme( 'block-theme' );
```

Comment:
Instead of switching theme on every function, I think we can do:
```
	public function set_up() {
		parent::set_up();
		switch_theme( 'block-theme' );
	}
```

And on the functions requiring another theme we just call switch_theme( other ); at the start.